test/util/testnode/config.go

```diff
@@ -119,7 +124,8 @@ func DefaultConfig() *Config {
 		WithAppConfig(DefaultAppConfig()).
 		WithAppOptions(DefaultAppOptions()).
 		WithAppCreator(cmd.NewAppServer).
-		WithSupressLogs(true)
+		WithSuppressLogs(true).
+		WithConsensusParams(DefaultConsensusParams())
```

Comment:
setting the default consensus params here fixes the failing test, which apparently we aren't doing doing on main! This doesn't fail on main, because we don't actually have a test that checks that the version is updated but now we do [here](https://github.com/celestiaorg/celestia-app/blob/50bc61ea72ddac7f7372ce7e4471be295670772b/test/util/testnode/full_node_test.go#L129-L136)